_data/releases.yml

```diff
@@ -0,0 +1,18 @@
+#Release versions
+
+legacy:
+  version: "0.0.0"
+  date: "YYYY-MM-DD"
+stable:
+  version: "0.1.0"
+  date: "2020-06-22"
+nightly:
+  version: "0.2-STABLE"
+  date: "2020-09-04"
```

Comment:
Similar is this required question here.  IMHO a date doesn't make sense for a nightly build.